Website/Website/Form/VisitorLogin.aspx

```diff
@@ -4,19 +4,52 @@
 
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head runat="server">
-    <title></title>
+    <title>Inloggen als Bezoeker</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#160e2f">
+    <link rel="icon" type="image/png" href="~/Resources/favicon.png">
+    <link href="~/Resources/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
+    <link href="~/Resources/css/bootstrap_overrides.css" rel="stylesheet" type="text/css" />
+    <link href="~/Resources/css/style.css" rel="stylesheet" type="text/css" />
+    <link href="~/Resources/css/style_visitor.css" rel="stylesheet" type="text/css" />
 </head>
```

Comment:
dit allemaal zou in principe in een master page kunnen.. maar misschien kost dit net iets te veel tijd om uit te vogelen..